extensions/amp-ad-network-doubleclick-impl/0.1/safeframe-host.js

```diff
@@ -219,6 +221,25 @@ export class SafeframeHostApi {
     return attributes;
   }
 
+  /**
+   * Returns the canonical URL of the page, if the publisher allows
+   * it to be passed.
+   * @return {string|boolean}
```

Comment:
Why not return empty string or undefined if not valid?  Boolean seems unnecessary

Reply:
done